raw-cms-app/src/modules/core/views/user-details-view/user-details-view.js

```diff
@@ -31,6 +32,8 @@ const _UserDetailsView = async (res, rej) => {
         this.title = isNew
```

Comment:
No need for `title` property on component, see comment on "raw-cms-app/src/modules/core/views/collection-table-view/collection-table-view.js".